lib/bloc/menubloc/menu_bloc.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:bloc/bloc.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:nutrilab/bloc/menubloc/menu_event.dart';
+import 'package:nutrilab/bloc/menubloc/menu_state.dart';
+import 'package:nutrilab/models/menuitemmodel.dart';
+
+class MenuBloc extends Bloc<MenuEvent, MenuState> {
+  final String userId;
+
+  MenuBloc(this.userId) : super(MenuInitial()) {
+    on<LoadMenuItems>(_onLoadMenuItems);
+  }
+
+  Future<void> _onLoadMenuItems(
+      LoadMenuItems event, Emitter<MenuState> emit) async {
+    emit(MenuLoading());
+
+    try {
+      DocumentSnapshot userSnapshot =
+          await FirebaseFirestore.instance.collection('users').doc(userId).get();
```

Comment:
I think that it'd be better to load user data in separate bloc which responsible for profile information, but for this PR it's ok.

Reply:
So should I pass likedItems and cartItems into this bloc as parameters?